molecularnodes/entities/molecule/array.py

```diff
@@ -0,0 +1,52 @@
+import uuid
+from biotite.structure import AtomArray
+
+from .base import Molecule
+from .pdb import _comp_secondary_structure
+from ..base import EntityType
+
+
+class Array(Molecule):
```

Comment:
All the. logic is in a new Array Class analogous to the others. This has to do a lot less work though!